functional/ek-cert-use-ek_check_script/test.sh

```diff
@@ -51,12 +59,22 @@ _EOF"
     rlPhaseEnd
 
     rlPhaseStartTest "Add keylime agent and check genuine of TPM via ek_check_script option"
-        rlRun "keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u $AGENT_ID --allowlist allowlist.txt --exclude excludelist.txt -f excludelist.txt -c update"
-        rlRun "limeWaitForAgentStatus $AGENT_ID 'Get Quote'" 
+        rlRun -s "keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u $AGENT_ID --allowlist allowlist.txt --exclude excludelist.txt -f excludelist.txt -c update"
+        rlAssertGrep "AGENT_UUID=$AGENT_ID" $rlRun_LOG -E
+        rlAssertGrep "EK=-----BEGIN PUBLIC KEY-----" $rlRun_LOG -E
+        rlRun "limeWaitForAgentStatus $AGENT_ID 'Get Quote'"
```

Comment:
It would be good to add test for the other 3 variables. I know the content may vary but at least check that they are listed and the content is not empty. Something like `NAME=[^ ]+` or similar.